comp/api/api/def/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/DataDog/datadog-agent/comp/api/api/def
+
+go 1.21.0
```

Comment:
We need to make sure we update the Go version if the [PR](https://github.com/DataDog/datadog-agent/pull/26488) to bump Go gets merged before this one.

Reply:
it'd be easiest to pull these changes into that branch since it has the buildimages for 1.22 I think